app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/UnauthorisedController.scala

```diff
@@ -32,7 +32,7 @@ import scala.concurrent.{ExecutionContext, Future}
 class UnauthorisedController @Inject() (
   val authConnector: AuthConnector,
   authRedirect: AuthRedirectSupport,
-  notSubscribedToCds: NotSubscribedToCds
+  notSubscribedToCds: not_subscribed_to_cds_page
```

Comment:
convention is:

Twirl template: `not_subscribed_to_cds` (ie not page suffix - we already know they are pages as they are twirl templates)
Reference name : `notSubscribedToCdsPage` (need page here as this will be reference in the implementation and it's helpful to know what that variable is ie it's a reference to an actual twirl template)